frontend/src/routes/FacilityBooking/CreateBooking/index.tsx

```diff
@@ -309,6 +310,7 @@ export default function CreateBooking() {
             setValue={setDescription}
             textarea
           />
+          <ConflictAlert />
```

Comment:
i think for now not necessary to include in the page, we'll redo the page soon

Reply:
Removed in commit e78c536